### DISCUSSION TITLE
About TI simulation

### DISCUSSION THREAD

**Discussion by lingoodluck** [simnibs/simnibs#474] [9. Results and analysis]
Hello! When I was trying the TI simulation, I placed two pairs of electrodes at F5-P5 and F6-P6 respectively. I looked at the simulation results of one pair of tDCS and found that the electric field at the compact bone suddenly dropped significantly in a triangular area on the other side. Can someone help explain this? Figure 1 shows the simulation results of one pair of tDCS, and Figure 2 shows the results of the TI simulation.
![图片3](https://github.com/user-attachments/assets/3ef5b837-b4c8-460c-b207-1352d57ff321)
![图片2](https://github.com/user-attachments/assets/b7c70b5a-8e6e-4491-8ab4-a0031eccd93b)

**Comment by TorgeW** [comment]
This seams to be a scaling issue of the visualization.
please find an explanation of how to change the scale here: https://simnibs.github.io/simnibs/build/html/tutorial/visualization.html
If you want to look at the field distributions at the scalp its useful to set the maximum to the actual maximum.
